### ISSUE TITLE
Build warnings: 'enableAndroidTest: Boolean' is deprecated

### ISSUE THREAD

**Issue by dturner** [android/nowinandroid#1272]
```
> Task :build-logic:convention:compileKotlin
w: file:///Users/donturner/Code/workspace-android/nowinandroid/build-logic/convention/src/main/kotlin/com/google/samples/apps/nowinandroid/AndroidInstrumentedTests.kt:33:8 'enableAndroidTest: Boolean' is deprecated. replaced with enableAndroidTest
w: file:///Users/donturner/Code/workspace-android/nowinandroid/build-logic/convention/src/main/kotlin/com/google/samples/apps/nowinandroid/AndroidInstrumentedTests.kt:33:31 'enableAndroidTest: Boolean' is deprecated. replaced with enableAndroidTest
```

**Comment by sanao1006** [comment]
I would like to work on this issue!

**Comment by Sivyamkumar** [comment]
Hey @dturner i would like  to know that if by any chance this issue is still open as i would like to contribute to your awesome projects. I am happy to work on this.

**Comment by sannysinghs** [comment]
Bit difficult to understand what is being request for change. If the change request is to build a lint warning for such usage or request to replace deprecated usage ?

**Comment by Jaehwa-Noh** [comment]
This issue had been resolved at AGP 8.4.0, no longer show warning.

**Comment by chaitnayah** [comment]
Hey @dturner I would like to work on this issue ,as a beginner to the open source contribution. I am happy to learn and work on this projects
